src/ClientModel/CaptureData.cpp

```diff
@@ -111,6 +111,37 @@ void CaptureData::UpdateFunctionStats(uint64_t instrumented_function_id, uint64_
   }
 }
 
+void CaptureData::OnCaptureComplete(std::vector<orbit_client_data::TimerChain*> chains) {
+  // Recalculate standard deviation as the running calculation may have introduced error.
+  for (auto& pair : functions_stats_) {
+    FunctionStats& stats = pair.second;
+    stats.set_variance_ns(0);
+  }
+
+  for (auto& chain : chains) {
+    if (!chain) continue;
+    for (const auto& block : *chain) {
+      for (uint64_t i = 0; i < block.size(); i++) {
+        const orbit_client_protos::TimerInfo& timer_info = block[i];
+        auto& stats = functions_stats_[timer_info.function_id()];
+        if (stats.count() > 0) {
+          uint64_t elapsed_nanos = timer_info.end() - timer_info.start();
+          uint64_t diff = elapsed_nanos - stats.average_time_ns();
```

Comment:
If elapsed_nanos < stats.average_time_ns(), there is an underflow here, no?

Reply:
nit. I think the "diff" term is the "deviation" https://en.wikipedia.org/wiki/Deviation_(statistics)

Reply:
yeah..I guess it still works because I immediately square it, but I changed it to an int (and renamed deviation).